### PR TITLE
Actual day6 dailyChallenge Push

### PR DESCRIPTION
**Pull request by sw00z**

### CODE REVIEW

File: codingtovictory/day6/dailyChallenge/day6DailyChallenge.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="stylesheet.css">
+</head>
+<body>
+
+    <h1>CHECKERBOARD</h1>
+    <div class="container">
+        <div class="row">
+            <div class="square black"></div>
```
Comment (OnkurLal) [inline codingtovictory/day6/dailyChallenge/day6DailyChallenge.html:14]: Nice job figuring out the checker board.

File: codingtovictory/day6/dailyChallenge/stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  
+  .row {
+    clear: both;
+  }
+
+h1 {
+    text-align: center;
+}
+
+ol {
```
Comment (OnkurLal) [inline codingtovictory/day6/dailyChallenge/stylesheet.css:28]: Nice one on figuring out the Roman Numerals.

**Review by OnkurLal — COMMENTED** [review]
All in all you did a great job! Also explore using rem for size instead of px,  down the line it will be better to use rem for responsiveness. Adjust the size of your window with both options and see the differences

**Review by OnkurLal — APPROVED** [review]

File: codingtovictory/day6/dailyChallenge/day6DailyChallenge.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+            <div class="square black"></div>
+            <div class="square red"></div>
+        </div>
+
+
+    <a href="https://www.pexels.com/photo/city-hotel-building-office-15903178/"><img src="Pics/pictures" style="height: 500px;width: 500px;padding: 150px;"></a>
```
Comment (victorSauceda) [inline codingtovictory/day6/dailyChallenge/day6DailyChallenge.html:81]: Try to stay await from inline styling and move these properties to your style sheet. Totally ok to do this but best practice is to decouple. Keep HTML with HTML and CSS with CSS. @OnkurLal @Bs-Sec

File: codingtovictory/day6/dailyChallenge/Pics/pictures
Comment (victorSauceda) [inline codingtovictory/day6/dailyChallenge/Pics/pictures:1]: What is the pictures file? @Bs-Sec . Are we using this in your project? Should have a file extension like `.png` or `.jpg`

**Review by victorSauceda — APPROVED** [review]

**Pull request merged**